algorithms/Path/maxflow/tg_maxflow.gsql

```diff
@@ -0,0 +1,186 @@
+CREATE QUERY tg_maxflow(VERTEX source, VERTEX sink, Set<STRING> v_type, SET<STRING> e_type, 
+       SET<STRING> reverse_e_type, STRING cap_attr, STRING cap_type, FLOAT min_flow_threshhold = 0.001, 
+       BOOL print_accum = TRUE, BOOL display_edges = TRUE, BOOL spit_to_file = FALSE, 
```

Comment:
parameter `spit_to_file` was never used in the query
please remove it if not needed